impeller/tools/build_metal_library.py

```diff
@@ -100,15 +100,19 @@ def main():
         '-Oz',
         # Allow aggressive, lossy floating-point optimizations.
         '-ffast-math',
+        # limiting to ios-metal1.2 disables shader debug symbols, only
+        # enabling these in optimize mode.
+        # see https://github.com/flutter/flutter/issues/106066
+        '--std=ios-metal1.2',
     ]
     if args.platform == 'ios':
-      # limiting to ios-metal1.2 disables shader debug symbols, only
-      # enabling these in optimize mode.
-      # see https://github.com/flutter/flutter/issues/106066
       command += [
-          '--std=ios-metal1.2',
           '-mios-version-min=10.0',
```

Comment:
Should this also be 11.0?

Reply:
Yeah, may I change it in a later patch though? It is unrelated to the sim fix.

Reply:
I also want to remove the workaround around optimizations that were making Xcode crashy. This script is looking a bit confusing now.